testgen/src/vote.rs

```diff
@@ -86,10 +85,21 @@ impl Generator<vote::Vote> for Vote {
         let block_id = block::Id::new(block_header.hash(), None);
         let validator_index = match self.index {
             Some(i) => i,
-            None => match header.validators.as_ref().unwrap().iter().position(|v| *v == *validator) {
+            None => match header
+                .validators
+                .as_ref()
+                .unwrap()
+                .iter()
+                .position(|v| *v == *validator)
+            {
```

Comment:
Can we extract this long chained call into its own variable for better readability?

Reply:
thanks:) done